translation/templates.rst

```diff
@@ -40,6 +40,11 @@ works when you use a placeholder following the ``%var%`` pattern.
     If you need to use the percent character (``%``) in a string, escape it by
     doubling it: ``{% trans %}Percent: %percent%%%{% endtrans %}``
 
+.. versionchanged:: 4.2
```

Comment:
cc @javiereguiluz can we use it?

Reply:
It won't be styled, etc. Maybe we can reuse `versionadded:: 4.2` because the new behavior was "added" in 4.2. Would that work for you?